cmd/ipfs/init.go

```diff
@@ -19,10 +19,13 @@ import (
 	cmds "github.com/ipfs/go-ipfs-cmds"
 	config "github.com/ipfs/go-ipfs-config"
 	files "github.com/ipfs/go-ipfs-files"
+	options "github.com/ipfs/interface-go-ipfs-core/options"
 )
 
 const (
 	nBitsForKeypairDefault = 2048
+	algorithmDefault       = options.Ed25519Key
```

Comment:
Since we're going to separate out the default changes from the initialization the default should be RSA here.